sessionctx/binloginfo/binloginfo.go

```diff
@@ -14,45 +14,49 @@
 package binloginfo
 
 import (
+	"io/ioutil"
 	"regexp"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
 
 	"github.com/juju/errors"
+	"github.com/pingcap/tidb-tools/tidb-binlog/node"
+	pumpcli "github.com/pingcap/tidb-tools/tidb-binlog/pump_client"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/metrics"
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/terror"
 	binlog "github.com/pingcap/tipb/go-binlog"
 	log "github.com/sirupsen/logrus"
-	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 )
 
 func init() {
 	grpc.EnableTracing = false
+	// don't need output pumps client's log
+	pumpcli.Logger.Out = ioutil.Discard
```

Comment:
why?

Reply:
at the first, zhou sir don't want pump client print any log, it may disturb tidb's log. so I add this code.
we can refine pump client's log later, and remove this line if any other reviewer agree.

Reply:
hard to debug, create an issue for it

Reply:
OK，I will create a issue